lib/class-wp-duotone-gutenberg.php

```diff
@@ -201,6 +201,43 @@ public static function output_footer_assets() {
 		}
 	}
 
+	/**
+	 * Adds the duotone SVGs and CSS custom properties to the editor settings so
+	 * they can be pulled in by the EditorStyles component in JS and rendered in
+	 * the post editor.
+	 *
+	 * @param array $settings The block editor settings from the `block_editor_settings_all` filter.
+	 * @return array The editor settings with duotone SVGs and CSS custom properties.
+	 */
+	public static function add_editor_settings( $settings ) {
+		$assets  = '';
+		$presets = 'body{';
```

Comment:
Tiniest of nits: would it be worth calling this variable `$duotone_presets` to distinguish it from other presets in the codebase? If someone's searching for where presets get added to block editor settings, it might help to have a bit more differentiation.